container_workflow_tool/distgit.py

```diff
@@ -138,6 +138,37 @@ def _set_from(self, fdata, from_tag):
                      f"FROM {imagename_without_tag}:{from_tag}\n", fdata)
         return ret
 
+    def _set_yaml_variable(self, fdata: str = "", tag: str = "", tag_str: str = "", variable: str = ""):
+        """
+        Updates test-openshift.yaml string
```

Comment:
just a insignificant thing, but it seems to me, that this comment is misleading, as this method is quite generic, and it can be applied to any yaml, not just test-openshift.yaml, isnt it?